src/rc_allocator.c

```diff
@@ -473,10 +473,6 @@ rc_allocator_mount(rc_allocator        *al,
          al->stats.curr_allocated++;
       }
    }
-   platform_default_log(
```

Comment:
I could go either way about deleting these messages. But, ... With reworked fix for PR #487 where we will default to `/dev/ null/` for `stdout`, it should not really matter that these info messages coming out, right?

What's your overall direction in such cases -- Should we all just not add any info-messages at all?

(I do find some info-messages quite useful and am personally more tolerant of messages.)

Reply:
I don't really know the answer, but these ones in particular aren't super useful. My question back is what sorts of notifications are actually useful?

There are places where we assert no space leaks, and that feels like the right way to monitor this sort of thing.